### ISSUE TITLE
[iOS] 'framework not found Lookback' and 'setupWithAppToken is undefined'

### ISSUE THREAD

**Issue by elimydlarz** [negativetwelve/react-native-lookback#21]
I should mention that I'm using React Native 0.40. I've installed react-native-lookback, run a pod install and a react-native link. But attempting to build the app in XCode results in the following error:

```
ld: framework not found Lookback
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

When running the app in a simulator using the React Native CLI, this happens

![image](https://cloud.githubusercontent.com/assets/785427/22450923/3c532242-e7be-11e6-8211-d57adc8152d5.png)

Right now, my Podfile looks like this.

```
platform :ios, '9.3'

target 'FieldOps' do
  use_frameworks!

  pod "LookbackSafe", "~> 1.4.1"

  target 'FieldOpsTests' do
    inherit! :search_paths
  end

end
```

**Comment by negativetwelve** [comment]
Hi @elimydlarz, thanks for reporting in. I haven't tested this library with Swift (I'm assuming you're using Swift because of `use_frameworks!`)

What version of Cocoapods are you using?

Here are some links that I found that might be causing this issue:

* https://github.com/CocoaPods/CocoaPods/issues/4605#issuecomment-275291022
* https://github.com/BranchMetrics/react-native-branch-deep-linking/pull/61

If any of these help, please let me know

**Comment by elimydlarz** [comment]
Thanks for your help!

I'm not sure exactly what you mean by 'using Swift' - I'm using React Native and haven't written any native code. I tried using and not using `use_frameworks!` and it had no effect.

I'm using Cocoapods 1.2.

**Comment by negativetwelve** [comment]
ahh okay, I misunderstood, this is a brand new project generated with `react-native-cli`?

I haven't updated our version of react-native to 0.40 yet so I'll have to spend some time doing that. When I first made this package, I had a lot of trouble linking up the headers correctly on the native side (which is similar to what you're having issues with)

**Comment by elimydlarz** [comment]
Yes, it's a react-native-cli generated project that we've been working on for a couple of months now. Are you thinking of updating this to work with React Native 0.40? 

We'd like to have this working by Wednesday and I might take a shot at it myself if you're not planning to get to it, but I don't like my odds =/ Do you have any suggestions on where to start?

**Comment by negativetwelve** [comment]
Unfortunately, I'm not going to have time to complete this by then. If you submit a PR, I'd be more than happy to merge that in and push out a new release.

The problem is the native headers were moved in RN 0.40: https://github.com/facebook/react-native/releases/tag/v0.40.0

It should just be a matter of changing the native headers and then it should be good to go

**Comment by negativetwelve** [comment]
hi @elimydlarz, just published 1.3.0 to npm with changes made by @distributedlife that should address your issues. please let me know if things are still not working!
